free-programming-books-id.md

```diff
@@ -109,6 +109,8 @@
 * [Tutorial Python](https://docs.python.org/id/3.8/tutorial/)
 * [Workshop Python 101](http://sakti.github.io/python101/)
 
+
+
 ### Vue
```

Comment:
```suggestion
* [Workshop Python 101](http://sakti.github.io/python101/)


### Vue
```

See how now there are only 2 line between the last resource in Python section and Vue?

Reply:
@endrose here as well